Open_Foam/Post_Process.ipynb

```diff
@@ -4,129 +4,112 @@
     "\n",
+    "$$ C_D~=~\\frac{F_D}{\\frac{1}{2} . \\rho_{ref} . V_{ref} ^2 . A_{proj}}. $$\n",
     "\n",
-    "## Cite Source(s)\n",
-    "```\n",
-    "> For more details on the physics, theory and equation derivation please see chapter 9, section 9.2 of \"A Physical Introduction to Fluid Mechanics by Alexander J. Smits\" 2nd edition. [Download Book Here!](http://www.efluids.com/efluids/books/efluids_books.htm)\n",
-    "```\n",
+    "In this equation $F_D$, the total drag force due to pressure and viscous forces within the flow acting on the cylinder's surface in the streamwise direction, is normalized with the available momentum within the undisturbed incoming flow that could be exerted on the cylinder. The value of this momentum is calculated using $\\rho_{ref}$ as the reference density of the flow, which is the density of the flow. $V_{ref}$ as the undisturbed streamwise flow velocity in the inlet and $A_{proj}$, which is the projected area of the cylinder surface. For a two-dimensional cylinder the projected area is equal to the diameter of the cylinder. This is the area where pressure and viscus forces are acting on the cylinder in the streamwise direction. It should be noted that the pressure and viscous forces in the perpendicular direction to streamwise direction cancel each other. Hence, the forces and areas that they act on does not have any contribution to the drag force and coefficient.\n",
+    "\n",
+    "As discussed earlier in the Simulation section user can take advantage of pre-written functions and libraries to calculate and report drag force coefficient values. The \"forceCoeffs\" function, added to `controlDict` dictionary file located in `\\system` directory, takes \"cylinder\" walls as the patch and calculates the forces on in the defined directions. This was completely discussed in the Simulation section. Table 1 provides a summary of comparison between the present study and literature calculated drag coefficients values:\n",
+    "\n",
+    "| Tip Speed Ratio #  | $C_D$ Present Study| $C_D$ D. Stojković *et al.* (2002) | Relative Error [%]|\n",
+    "| :--------------------------------:|:------------------:|:--------------------------------:|:-----------------:|\n",
+    "| 0.5        | 1.2819      | 1.2740        |0.6|\n",
+    "| 1          | 1.1112      | 1.1080        |0.2|\n",
+    "| 1.5        | 0.8358      | 0.8180        |2.1|\n",
+    "| 2          | 0.5415      | 0.4611        |17.4|\n",
+    "Table 1. Comparison between the numerically predicted drag coefficient on a rotating cylinder with Re=100 and different Tip Speed Ratios.\n",
```

Comment:
Why only drag coefficient values comparison is presented here? Can we include comparison of lift coefficient, due to Magnus Effect, as well?